test/testNotificationsMultipleMachines.js

```diff
@@ -23,29 +25,38 @@ describe('mercurius (multiple-machines-)notify', function() {
   });
 
   it('sends notifications to multiple machines of a registered user', function(done) {
+    var count = 0;
+    function countReplies() {
+      count++;
+      return [201, 'boo'];
```

Comment:
What is 'boo'?

Instead of `countReplies`, you could keep the code as it was and add an assertion using the nock's `isDone` function, something like: `assert.true(nock.isDone(), 'test finished without pending requests: ' + nock.pendingMocks());`.